packages/apps/tools/src/utils/network.ts

```diff
@@ -89,6 +89,9 @@ export const getApiHost = ({
   if (!api.startsWith('http')) {
     scheme = 'https://';
   }
+  if (api.includes('localhost')) {
+    scheme = 'http://';
```

Comment:
What has changed that is now needed?

Reply:
if the user enters only `localhost:8080` as a value in the field '`network api`' when adding new network in will prefix it with '`https://`' and I wanted to handle this case

Reply:
Alright 👍